ecs-deploy

```diff
@@ -270,6 +286,8 @@ function getCurrentTaskDefinition() {
       # Get current task definition name from service
       TASK_DEFINITION_ARN=`$AWS_ECS describe-services --services $SERVICE --cluster $CLUSTER | jq -r .services[0].taskDefinition`
       TASK_DEFINITION=`$AWS_ECS describe-task-definition --task-def $TASK_DEFINITION_ARN`
+      
+      printf %s "$TASK_DEFINITION_ARN" > currenttaskdef
```

Comment:
I tried saving this value in an env variable but it didn't work, so I ended up using a temp file.

Reply:
Have you tried `export`? I think it could work. But the file approach also works fine.

Reply:
Yup I have. Apparently, this is an expected behavior according to [this](https://stackoverflow.com/questions/36394295/sharing-variables-between-two-independent-shell-scripts)